.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Lol, do you have to use frustrated here..

Reply:
i copied from template from github

Reply:
Yes, that's mandatory! It sets the proper tone and leads to really great issues. Just a few examples and you'll love it too:
https://github.com/nspcc-dev/neofs-node/issues/2647
https://github.com/nspcc-dev/neofs-node/issues/2613
https://github.com/nspcc-dev/neofs-node/issues/2579
https://github.com/nspcc-dev/neofs-node/issues/2497

Reply:
#356 added it back